Resources/views/line/folder.html.twig

```diff
@@ -0,0 +1,7 @@
+{% if subLinks|length > 0 %}
+    <ul>
```

Comment:
If this is used instead of `Resources/views/line/line.html.twig` then you should follow the same setup, but this seems to be something else or?